docs/user/server-side-search/api.rst

```diff
@@ -18,6 +18,9 @@ API v3
    .. Request
 
    :query q: Search query (see :doc:`/server-side-search/syntax`)
+   :query project: Project slug to be used when a query includes the ``@this`` alias.
+   :query version: Version slug to be used when a query includes the ``@this`` alias,
+    If only the project is given, ``@this`` will use the default version of the project.
```

Comment:
we could be more explict and call these `current-project` and `current-version`, but I'm fine with just project and version.